manifests/argocd-server-patch/kustomization.yaml

```diff
@@ -1,9 +1,5 @@
 apiVersion: kustomize.config.k8s.io/v1alpha1
 kind: Component
 
-resources:
```

Comment:
I suggest to keep the current kustomization as is to avoid breaking users.

Reply:
I reinstate the kustomize to the original state to avoid any existing users. However, I had to move the patch file under a different folder `argocd-server-patch`  as customize doesn't allow to refer patches outside the folder. So, I have included the patch as components